docs/concepts/controller-manager.md

```diff
@@ -107,6 +107,25 @@ The `--event-ttl` should be larger than the `ttlNonShootEvents` or this controll
 
 Consequently, to ensure that `ExposureClass`s in-use are always present in the system until the last referring `Shoot` gets deleted, the controller adds a finalizer which is only released when there is no `Shoot` referencing the `ExposureClass` anymore.
 
+### [`ManagedSeedSet` Controller](../../pkg/controllermanager/controller/managedseedset)
+
+`ManagedSeedSet` objects maintain a stable set of replica of `ManagedSeed`s, i.e they guarantee the availability of a specified number of identical `ManagedSeed`s on an equal number of identical `Shoot`s.
+`ManagedSeedSet` controller creates and deletes `ManagedSeed`s and `Shoot`s in response to changes to the replicas and selector fields. For more info refer to the [`ManagedSeedSet` proposal document](../../docs/proposals/13-automated-seed-management.md#managedseedsets).
+
+1. The reconciler first gets all the replicas of the given `ManagedSeedSet` in the `ManagedSeedSet`'s namespace and with the matching selector. Each replica is a tuple that contains a `ManagedSeed`, its corresponding `Seed` and `Shoot` objects.
```

Comment:
What do you mean by "each replica is a tuple"? Maybe remove this or use "group" or "structure" instead?